model.py

```diff
@@ -1,23 +1,24 @@
 from dao import Dao
-
-collName='students'
-
+import sys
 class Model:
-
-    #def save
-    #def get_by_params
   
     def __init__(self, collection):
-        self.id = collection['id'] if 'id' in collection else None
+        self._id = str(collection['_id'])
```

Comment:
always check that the key you are trying to access within a collection exists before trying to access it ... otherwise a KeyError be thrown and cause the application to crash ...
```py
self._id = str(collection['_id']) if '_id' in collection else None
```